src/commands/ping.ts

```diff
@@ -13,29 +13,34 @@ module.exports = new Command({
 		{ name: 'streamer', required: true },
 	],
 	allowedChannels: ['bot-commands'],
-	async execute({ msg, args, mongoLobbies }) {
+	async execute({ ia, mongoLobbies }) {
 		// Validate
-		const pingsChannel = msg.guild.channels.cache.find((c) => c.name === 'player-pings') as TextChannel
-		if (!pingsChannel) throw new ClientError(msg, 'Channel player-pings does not exist')
+		const pingsChannel = ia.guild.channels.cache.find((c) => c.name === 'player-pings') as TextChannel
+		if (!pingsChannel) throw new ClientError(ia, 'Channel player-pings does not exist')
+
+		if (ia.options.data.length < 3) throw new ClientError(ia, 'Invalid number of arguments. Format is "!ping <rank> <region> <streamer>')
+		let rankPing = ia.options.data[0].value.toString()
+		let regionPing = ia.options.data[1].value.toString()
+		let streamerPing = ia.options.data[2].value.toString()
```

Comment:
I would switch the words, because they sound like individual pings, or remove the "Ping" all together